src/main/java/org/cryptomator/ui/fxapp/UpdateCheckerModule.java

```diff
@@ -20,8 +20,11 @@
 import java.net.URI;
 import java.net.http.HttpClient;
 import java.net.http.HttpRequest;
+import java.time.temporal.ChronoUnit;
+import java.time.temporal.TemporalUnit;
 import java.util.Optional;
 import java.util.concurrent.ExecutorService;
+import java.util.concurrent.TimeUnit;
```

Comment:
Are all these imports used?

Reply:
Fixed in https://github.com/cryptomator/cryptomator/pull/3157/commits/3567c036c3842c2092d5e4c0b6421bb65cc315db